recipes/vc-feature/meta.yaml

```diff
@@ -0,0 +1,23 @@
+{% set version = "0.0.1" %}
```

Comment:
I think the versioning should actually match vc. So, we would use selectors to determine the version.